lib/AbstractReader.js

```diff
@@ -21,17 +21,17 @@ class AbstractReader {
 	}
 
 	/**
-	 * Locates resourcess by GLOB.
+	 * Locates resources by matching GLOB patterns.
```

Comment:
I would also use "glob" not "GLOB". At least that's how it's spelled everywhere: https://en.wikipedia.org/wiki/Glob_(programming)

Reply:
Done